packages/blade/src/storybook-recipes/ThemePlayground/index.tsx

```diff
@@ -37,8 +38,11 @@ const ThemePlayground = (): React.ReactElement => {
         border: {
           radius: {
             none: 0,
+            // @ts-ignore
```

Comment:
why are we adding these to storybook now? was it always throwing error that wasn't captured earlier?

Reply:
Yes it was throwing some weird errors suddenly.